### PR TITLE
Adding GCP support

### PR DESCRIPTION
**Pull request by fao89**
https://github.com/jschneier/django-storages/pull/1193

Required PR: https://github.com/pulp/pulpcore/pull/3367

No Issue! Just a draft PR to have an idea on what is needed for GCP support

### CODE REVIEW

File: pulp_container/app/redirects.py [hunk 1]
```diff
@@ -132,3 +132,21 @@ def redirect_to_object_storage(self, artifact, return_media_type):
...
+
+
+class GCloudStorageRedirects(S3StorageRedirects):
+    """
+    A class that implements methods for the direct retrieval of manifest objects.
+    """
+
+    def redirect_to_object_storage(self, artifact, return_media_type):
+        """
+        Redirect to the passed artifact's file stored in the Azure storage.
```
Comment (ipanova) [inline pulp_container/app/redirects.py:144]: s/Azure/GCP

**Review by ipanova — APPROVED** [review]
looks good to me, please once django-storages releases and pulpco PR is merged undraft this one and merge it. thanks!
And this needs issue please + changelog entry

File: pulp_container/app/redirects.py (hunk 1 above)
Comment (mdellweg) [reply pulp_container/app/redirects.py:144]: I think we have a pulpcore function for this now.

**Comment by fao89** [comment]
it was just a draft

**Comment by ipanova** [comment]
why closing it if the code does not need any changes?

**Comment by fao89** [comment]
without CI I don't know if it needs more changes, better to start a PR when we have a CI job for it

**Pull request closed without merging**